docs/developers/guides/run-linea/run-the-stack/local.mdx

```diff
@@ -0,0 +1,115 @@
+---
+description: Run the Linea stack locally.
+image: /img/socialCards/run-the-linea-stack-locally.jpg
+---
+
+import Tabs from "@theme/Tabs";
+import TabItem from "@theme/TabItem";
+
+# Run locally
+
+## Prerequisites
+
+The following installed:
+
+- [Node.js v18](https://nodejs.org/en/download/package-manager)
```

Comment:
```suggestion
- [Node.js v18+](https://nodejs.org/en/download/package-manager)
```

Reply:
"or later" for consistency with the other list items